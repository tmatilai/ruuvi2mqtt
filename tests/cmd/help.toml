bin.name = "ruuvi2mqtt"
args = "--help"
