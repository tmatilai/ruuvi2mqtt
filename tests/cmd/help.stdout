Usage: ruuvi2mqtt [OPTIONS]

Options:
      --config <CONFIG>        Configuration file [env: CONFIG_FILE=] [default: ruuvi2mqtt.yaml]
      --log-level <LOG_LEVEL>  [env: LOG_LEVEL=] [default: INFO]
  -h, --help                   Print help information
  -V, --version                Print version information
